Boolean-Believers/nutshell
# Created the file structure and created the event HTML form that will …

# Description

I have created the file structure (modules) for the events. Also, in the formBuilderAndEditor.js module, I created three functions:

1) newEventButton
2) newEventFormBuilder
3) editEventFormBuilder

In the console, you should be able to see these functions as objects within the eventsFormBuilder variable.

As a side note, I commented out a lot of our console logs to avoid having a bunch of stuff in the console (we could consider removing them).

## Type of change


# Testing Instructions

- git fetch --all
- git checkout event-list-build-form
- Check to see if the code in formBuilderAndEditor.js makes sense and looks good
- Also, check that that info shows up in the console

# Checklist:
